lib/Imagine/Imagick/Image.php

```diff
@@ -858,6 +867,34 @@ private function detectColorspaceConversionSupport()
         return static::$supportsColorspaceConversion = method_exists('Imagick', 'setColorspace');
     }
 
+    /**
+     * ImageMagick without the lcms delegate cannot handle profiles well.
+     * This detection is needed because there is no way to directly check for lcms.
+     *
+     * @return Boolean
+     */
+    private function detectProfilesSupport()
+    {
+        if (null !== static::$supportsProfiles) {
```

Comment:
I'd use `self` instead of `static` because everything is private. But it's just my personal taste: feel absolutely free to ignore this comment from me :wink:

Reply:
Makes sense. Private members should be accessed via `self::`, done in ec96c8a463e18b81d0537414ae25604e0225595d